src/messages.rs

```diff
@@ -55,3 +55,8 @@ pub use user_notice::{UserNotice, UserNoticeId};
 
 mod twitch_message;
 pub use self::twitch_message::TwitchMessage;
+
+pub use twitch_types::{
+    self as types, BadgeSetIdRef, ChatBadgeIdRef, DisplayNameRef, EmoteIdRef, EmoteSetIdRef,
+    IntoCow, MsgIdRef, UserIdRef, UserNameRef, UserType,
+};
```

Comment:
should this be re-exported like this? or should it just refer to the extern crate and re-export the crate at the root?

*edit* we can decide before the publish